Nef_2/include/CGAL/Nef_2/PM_io_parser.h

```diff
@@ -313,13 +340,26 @@ template <typename PMDEC>
 void PM_io_parser<PMDEC>::read()
 {
   if ( !check_sep("Plane_map_2") )
-    CGAL_error_msg("PM_io_parser::read: no embedded_PM header.");
+  {
+    std::cerr<<"PM_io_parser::read: no embedded_PM header."<<std::endl;
```

Comment:
why removing the CGAL macros?

Reply:
They call std::abort and crash the program.

Reply:
Except if there is a verbose mode, nothing should be written to std::cerr. If there is no verbose mode maybe CGAL_warning_msg can be used.